### DISCUSSION TITLE
Problema con MVN

### DISCUSSION THREAD

**Discussion by camilamarinio** [dds-jv/foro#245] [Consultas de herramientas] [answered]
Hola , cuando corro mvn test me tira este error en el tp de archivos :

![Captura de pantalla 2021-05-27 211816](https://user-images.githubusercontent.com/62487974/119912124-321d0f00-bf31-11eb-936a-42558f288bb1.png)
Como lo puedo solucionar?

**Comment by nanderson87** [comment, marked as answer]
Hola @camilamarinio 

Por lo que se ve en el error que adjuntaste parece un error sintáctico. Imagino que quisiste ejecutar el comando `mvn test ` y terminaste escribiendo `tets`. Puede ser?

**Reply by gastonprieto** [reply]
Funciono @camilamarinio ? Si te funciono marca la respuesta como respondida
